news/2017-10-13-osu-world-cup-2017-registrations-now-open.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "osu! World Cup 2017: Registrations now open!"
+date: 2017-10-13 13:00:00 +0000
+---
```

Comment:
With a brand new design, osu!'s fiercest and most exciting competition is just getting started!

Reply:
fixed